src/de.hpi.swa.trufflesqueak.launcher/src/de/hpi/swa/trufflesqueak/launcher/TruffleSqueakLauncher.java

```diff
@@ -67,6 +67,8 @@ protected List<String> preprocessArguments(final List<String> arguments, final M
                 final List<String> remainingArguments = arguments.subList(i + 1, arguments.size());
                 imageArguments = remainingArguments.toArray(new String[0]);
                 break;
+            } else if (arg.equals("xxx")) {
```

Comment:
This seems to be some sort of `--delete-image` option, right? Might be useful, although deleting files is a bit dangerous. Maybe you could list all files/directories first, then ask for user confirmation?

Reply:
BTW: I typically just point TruffleSqueak at different images during development (e.g., `trufflesqueak images/cuis.image` and `trufflesqueak images/squeak.image`).